### PR TITLE
Update hello-hybrid-cloud-kerberos-trust-provision.md

### PR DESCRIPTION
**Pull request by bartbilliet**
Added clarification concerning HAADJ vs AADJ with regards to cloud Kerberos trust.

## Why
Line-of-sight to domain controller requirement for Hybrid Azure AD joined devices was clearly described, however the difference with regards to Azure AD Join & line-of-sight to DC could be more clear. I wrote a proposal to make this difference more clearly articulated. Feel free to propose changes if my statement would be incorrect.

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cloud-kerberos-trust-provision.md [hunk 1]
```diff
@@ -177,9 +177,9 @@ If you deployed Windows Hello for Business using the key trust model, and want t
 1. For hybrid Azure AD joined devices, sign out and sign in to the device using Windows Hello for Business.
 
 > [!NOTE]
-> For hybrid Azure AD joined devices, users must perform the first sign in with new credentials while having line of sight to a DC.
->
-> Without line of sight to a DC, even when the client is configured to use cloud Kerberos trust, the system will fall back to key trust if cloud Kerberos trust login fails.
+> For hybrid Azure AD joined devices, users must perform the first sign in with new credentials while having line of sight to a DC. Without line of sight to a DC, even when the client is configured to use cloud Kerberos trust, the system will fall back to key trust if cloud Kerberos trust login fails.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cloud-kerberos-trust-provision.md:180]:
```suggestion
> For hybrid Azure AD joined devices, users must sign in the first time with new credentials while having a line of sight to a DC. Without line of sight to a DC, even when the client is configured to use cloud Kerberos trust, the system will fall back to key trust if signing in to cloud Kerberos trust fails.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cloud-kerberos-trust-provision.md [hunk 2]
```diff
@@ -177,9 +177,9 @@ If you deployed Windows Hello for Business using the key trust model, and want t
...
+> 
+> For Azure AD joined devices, a line of sight to a DC is not required during the first sign in with new credentials, and is only required in order to access on-premises resources.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cloud-kerberos-trust-provision.md:182]:
```suggestion
> For Azure AD joined devices, a line of sight to a DC is not required when first signing in with new credentials, and is only required in order to access on-premises resources. 
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cloud-kerberos-trust-provision.md [hunk 3]
```diff
@@ -196,6 +196,8 @@ If you deployed Windows Hello for Business using the certificate trust model, an
 
 > [!NOTE]
 > For hybrid Azure AD joined devices, users must perform the first sign-in with new credentials while having line of sight to a DC.
+> 
+> For Azure AD joined devices, a line of sight to a DC is not required during the first sign in with new credentials, and is only required in order to access on-premises resources.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cloud-kerberos-trust-provision.md:200]:
```suggestion
> For hybrid Azure AD joined devices, users must sign in the first time with new credentials while having line of sight to a DC.
> 
> For Azure AD joined devices, a line of sight to a DC is not required when first signing in with new credentials, and is only required in order to access on-premises resources.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bartbilliet, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by paolomatarazzo** [comment]
Hello @bartbilliet and thank you for the contribution. I feel that the proposed change adds confusion. You can't sign in to an Azure AD join devices using AD auth. The authentication can only be done by Azure AD (or a federated IdP). Therefore, adding the proposed statement is misleading.
The current statement is simple and technically correct, referencing only HAADJ devices.

**Comment by bartbilliet** [comment]
My mind went in this direction, since you can do SSO to on-prem resources from AADJ devices. That's why I started doubting if for AADJ devices you would need line-of-sight to an on-prem DC as well during WHfB enrollment.

**Comment by paolomatarazzo** [comment]
I'll revise the document to improve clarity, thank you for the feedback!

**Pull request closed without merging**